gooey/python_bindings/gooey_decorator.py

```diff
@@ -33,7 +33,10 @@ def Gooey(f=None,
           optional_cols=2,
           dump_build_config=False,
           load_build_config=None,
-          monospace_display=False):
+          monospace_display=False,
+          progress_regex=None,
+          progress_expr=None,
+          progress_animation=True):

```

Comment:
@chriskiehl Would you like just to hide stop button or hide it and move progress bar to the right of the footer (as it was before)?